compact/ecal/backward_PbWO4.xml

```diff
@@ -39,11 +49,20 @@
             r12max="EcalEndcapN_structure_ring_max"
             SFlength="EcalEndcapN_structure_ring_length"
             CMlength="EcalEndcapN_CrystalModule_length + 2.*EcalEndcapN_CrystalModule_wrap_thickness"
+            NEEMC_PR="EcalEndcapN_Protate"
+            NEEMC_NR="EcalEndcapN_Nrotate"
+            NEEMC_OR_RS="EcalEndcapN_Oring_relative_shift"
+            Inner_a="EcalEndcapN_Iring_axisA"
+            Inner_b="EcalEndcapN_Iring_axisB"
             envelope="true"
-            vis_struc="RPVis"
-            vis_steel_gap="AnlGray"
             ring_material="StainlessSteel"
-            sector="1">
+            ptsX_extrudedpolygon="6.15*cm 6.15*cm 8.2*cm 8.2*cm 6.15*cm 6.15*cm -6.15*cm -6.15*cm -8.2*cm -8.2*cm -6.15*cm -6.15*cm"
+            ptsY_extrudedpolygon="6.15*cm 4.1*cm 4.1*cm -4.1*cm -4.1*cm -6.15*cm -6.15*cm -4.1*cm -4.1*cm 4.1*cm 4.1*cm 6.15*cm"
+            inner_outer_add_posx="1.025 1.025 3.075 3.075 5.125 7.175 7.175 9.225 9.225 -1.025 -1.025 -3.075 -3.075 -5.125 -7.175 -7.175 -9.225 -9.225 -1.025 -1.025 -3.075 -3.075 -5.125 -7.175 -7.175 -9.225 -9.225 1.025 1.025 3.075 3.075 5.125 7.175 7.175 9.225 9.225 9.225 11.275 13.325 15.375 17.425 19.475 44.075 46.125 60.475 60.475 62.525 62.525 62.525 62.525 -9.225 -11.275 -13.325 -15.375 -17.425 -19.475 -44.075 -46.125 -60.475 -60.475 -62.525 -62.525 -62.525 -62.525 -9.225 -11.275 -13.325 -15.375 -17.425 -19.475 -44.075 -46.125 -60.475 -60.475 -62.525 -62.525 -62.525 -62.525 9.225 11.275 13.325 15.375 17.425 19.475 44.075 46.125 60.475 60.475 62.525 62.525 62.525 62.525"
+            inner_outer_add_posy="7.175 9.225 7.175 9.225 7.175 7.175 5.125 1.025 3.075 7.175 9.225 7.175 9.225 7.175 7.175 5.125 1.025 3.075 -7.175 -9.225 -7.175 -9.225 -7.175 -7.175 -5.125 -1.025 -3.075 -7.175 -9.225 -7.175 -9.225 -7.175 -7.175 -5.125 -1.025 -3.075 62.525 62.525 62.525 62.525 60.475 60.475 46.125 44.075 19.475 17.425 15.375 13.325 11.275 9.225 62.525 62.525 62.525 62.525 60.475 60.475 46.125 44.075 19.475 17.425 15.375 13.325 11.275 9.225 -62.525 -62.525 -62.525 -62.525 -60.475 -60.475 -46.125 -44.075 -19.475 -17.425 -15.375 -13.325 -11.275 -9.225 -62.525 -62.525 -62.525 -62.525 -60.475 -60.475 -46.125 -44.075 -19.475 -17.425 -15.375 -13.325 -11.275 -9.225"
```

Comment:
Keeping the x and y coordinates in separate lists is not ideal. The values that belong together are not aligned, and no one can figure out which of them fit together without significant effort. If you want to store arrays of structured info, you should store your arrays as structured info. Take a look at https://github.com/eic/epic/blob/main/compact/pid/mrich.xml for another detector that specifies lots of positions in the compact file.

Reply:
done by the last commit